src/modules/logger/logged_topics.cpp

```diff
@@ -57,6 +57,7 @@ void LoggedTopics::add_default_topics()
 	add_topic("commander_state");
 	add_topic("config_overrides");
 	add_topic("cpuload");
+	add_topic("differential_drive_setpoint", 100);
```

Comment:
```suggestion
	add_optional_topic("differential_drive_setpoint", 100);
```

Reply:
Thanks! When are we supposed to add_topic and when use add_optional_topic?

Reply:
Clear now, thanks @bkueng!